client/src/www/views/root_view/root_navigation/index.ts

```diff
@@ -0,0 +1,267 @@
+/*
+  Copyright 2024 The Outline Authors
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import {LitElement, html, css, nothing} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {classMap} from 'lit/directives/class-map.js';
+
+import navigationLogo from '../../../assets/logo-nav.png';
+
+@customElement('root-navigation')
+export class RootNavigation extends LitElement {
+  @property({type: Function}) localize!: (
```

Comment:
You can use `Localizer` from '@outline/infrastructure/i18n'. I also suggest setting a default function so you can avoid the non-nullability assertion, which is unsafe.

```suggestion
  @property({type: Function}) localize: Localizer = msg => msg;
```